eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -1909,6 +1909,20 @@
     <Method name="setContentBytes"/>
     <Bug pattern="EI_EXPOSE_REP2"/>
   </Match>
+  
+   <!-- Returning a new copy of the object is not necessary -->
+  <Match>
```

Comment:
@abhijitpai can you please take a look?

Reply:
This is the same as the ClientEncryptionKeyProperties class in .net, an instance of which is returned for read on the CEK and is used by the Encryption library - if the customer reads a CEK by themselves and changes the wrappedDEK byte array on it, it doesn't affect anything. In .net standard 2.1, we could have used ReadOnlyMemory<T> - if Java has similar constructs, they can be used, else the exception seems fine to me.